sponsors/models.py

```diff
@@ -969,6 +973,18 @@ class Meta(BaseTieredQuantity.Meta, BenefitFeatureConfiguration.Meta):
         verbose_name = "Tiered Benefit Configuration"
         verbose_name_plural = "Tiered Benefit Configurations"
 
+    @property
+    def benefit_feature_class(self):
+        return TieredQuantity
+
+    def get_benefit_feature(self, **kwargs):
+        if kwargs["sponsor_benefit"].sponsorship.level_name == self.package.name:
```

Comment:
thoughts on how to handle the fact that we don't keep a pointer to the source `SponsorshipPackage` around, @berinhard? This seems _OK_ since the package will almost certainly exist with the same name when the application is submitted... but still don't feel great about making it a string comparison.

Reply:
I felt the same after coding the `list_sponsors` template tag. I coded a [similar package name checking](https://github.com/python/pythondotorg/blob/main/sponsors/templatetags/sponsors.py#L38) and I wanted to talk about a refactoring as well.

I'm not sure why we made the decision of not pointing to the `SponsorshipPackage` instance initially but this condition exposes that we need to have it. I can work on a data migration and refactoring to add this to the `Sponsorship` model.

Reply:
This approach makes the most sense to me 👍🏼